wp-content/themes/core/components/comments/comment/Comment_Controller.php

```diff
@@ -120,4 +124,13 @@ public function get_time( $format = 'c' ): string {
 		return date( $format, get_comment_time( 'U' ) );
 	}
 
+	public function get_starting_markup_element(): string {
+		$element = 'li';
+		if ( 'div' === $this->style ) {
+			$element = 'div';
+		}
+
+		return $element;
```

Comment:
This block of code could potentially be reduced to a single line, like this: 

```suggestion
		return ( 'div' === $this->style ) ? 'div' : 'li';
```